ui_tests/src/test/resources/com/saucedemo/account/SignInNegative.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Sign In with Invalid data
+
+  Background: I open start page
+
```

Comment:
@masrur18 The `I open start page` step is not going to be executed in this way, the step should be added on the next line prefixed with one of the BDD keywords (Given, When, Then, And, But).
Please update it